src/business-logic/services/ingested-entity-to-entity-mapper.ts

```diff
@@ -60,6 +60,7 @@ export class IngestedEntityToEntityMapper {
       isNext: false,
       isUnsynced: false,
       expectedDurationInMs: ingestedSegment.budgetDuration,
+      definesShowStyleVariant: ingestedSegment.definesShowStyleVariant,
```

Comment:
To remove the optionality of the attribute on the `Segment` type, a default value of `false` could be specified here.

Reply:
It has been done so!